### PR TITLE
Update libtiff to version 4.3.0

### PR DESCRIPTION
**Pull request by saukko**
[libtiff] Update to version 4.3.0. Fixes JB#55134.

Signed-off-by: Marko Saukko <marko.saukko@jolla.com>

### CODE REVIEW

**Comment by mlehtima** [comment]
Change submodule to point to https://github.com/sailfishos-mirror/libtiff

File: rpm/libtiff.spec [hunk 1]
```diff
@@ -53,6 +53,7 @@ Man and info pages for %{name}.
 
 %build
 export CFLAGS="%{optflags} -fno-strict-aliasing"
+sed -i "s/for.*/for false/g" autogen.sh
```
Comment (mlehtima) [inline rpm/libtiff.spec]: This affects also https://github.com/sailfishos-mirror/libtiff/blob/v4.3.0/autogen.sh#L3 and https://github.com/sailfishos-mirror/libtiff/blob/v4.3.0/autogen.sh#L6 and breaks the build.

Comment (saukko) [reply rpm/libtiff.spec]: Fixed.

**Comment by saukko** [comment]
> Change submodule to point to https://github.com/sailfishos-mirror/libtiff

Fixed.

**Review by mlehtima — APPROVED** [review]

**Review by mkosola — APPROVED** [review]

**Pull request merged**